### DISCUSSION TITLE
Compiz Effect

### DISCUSSION THREAD

**Discussion by yashkathe** [Schneegans/Burn-My-Windows#455] [Ideas]
Can we have a new animation with compiz effect like MAC

**Comment by Schneegans** [comment]
What effect are you referring to? Compiz had a lot of effects and I am not aware of any window-close animation on macOS.

**Comment by yashkathe** [comment]
I am talking about the following effect:

[Screencast from 10-13-2024 07:00:59 PM.webm](https://github.com/user-attachments/assets/0c04c43d-7d56-477f-b2be-33b4ea5db745)

Sorry for not being specific earlier.

**Reply by Schneegans** [reply]
Then this is a duplicate of https://github.com/Schneegans/Burn-My-Windows/issues/299
